frontend/src/routes/Infrastructure/InfraEnvironments/Details/InfraEnvironmentDetailsPage.css

```diff
@@ -0,0 +1,8 @@
+.pf-c-dropdown__menu {
```

Comment:
This will affect any dropdown menus on the InfraEnv details page. Le'ts remove this and rely on the PatternFly behaviour. A different language will have different text length anyway.